src/_compress.pyx

```diff
@@ -18,7 +18,9 @@ cimport libc.stdio
 cimport openmp
 
 from libc.stdlib cimport malloc, free, realloc
-from libc.stdint cimport uint8_t, uint32_t
+ctypedef unsigned char  uint8_t
+ctypedef unsigned int   uint32_t
```

Comment:
Certainly:
http://linux.die.net/include/stdint.h
seem to agree with you - does this fix the build on Window?